sdk/src/test/java/com/mapbox/maps/MapboxMapTest.kt

```diff
@@ -664,6 +664,103 @@ class MapboxMapTest {
     }
   }
 
+  @Test
+  fun getClusterLeavesDefaultParam() {
+    val feature = mockk<Feature>()
+    val mapSlot = slot<HashMap<String, Value>>()
+    val callback = mockk<QueryFeatureExtensionCallback>()
+    mapboxMap.getGeoJsonClusterLeaves("id", feature, callback = callback)
+    verify {
+      nativeMap.queryFeatureExtensions(
+        "id",
+        feature,
+        "supercluster",
+        "leaves",
+        capture(mapSlot),
+        callback
+      )
+    }
+    checkCapturedMap(mapSlot, "10", "0")
```

Comment:
Maybe expose default values public and check here with a reference instead of hardcoded values?

Reply:
This is to verify the default values in MapboxMap are not be changed by accident. If we use the exposed default values, we can't such errors.

Reply:
agree with @yunikkk that making things as `QFE_SUPER_CLUSTER` internal instead of private could make this test code a bit cleaner but just a nit